### ISSUE TITLE
The chart-releaser GitHub Action is failing

### ISSUE THREAD

**Issue by morningspace** [IBM/cp4waiops-gitops#177]
See: https://github.com/IBM/cp4waiops-gitops/runs/7036836464 and https://github.com/IBM/cp4waiops-gitops/actions/runs/2553948296 as example

**Comment by morningspace** [comment]
This issue may be also related to https://github.com/IBM/cp4waiops-gitops/issues/136.

I removed all the tags and releases that are created unexpectedly, w/ following TODOs:
- Since we only officially support GitOps install started from v3.4, all the releases and tags we created previously can be dropped.
- The chart-releaser config can also be updated to only pick up v3.4 given the same reason as above.
- For the AIManager and EventManager release name, it was previously in the form of: `<eventmanager|aimanager><release version>-<chart version>`, e.g.: `eventmanager33-0.0.1`, `aimanager32-0.0.1`. How about we change it to something such as `eventmanager-3.4.0`, `aimanager-3.4.0`, where `3.4` represents the product release version, and the last number represents the helm chart revision number. Originally I thought it could be conflicted w/ the product fix pack release version, e.g. after CP4WAIOps 3.4.0, there will be FP release 3.4.1 coming soon. However, given that we do not have too much helm chart (i.e. GitOps configuration) change per each FP release, we can simplify it by only sticking to the product major release number, and use the last number as the helm chart revision to reflect the changes we made to the GitOps configuration itself.
